go.sum

```diff
@@ -1525,6 +1526,8 @@ github.com/prometheus/tsdb v0.8.0/go.mod h1:fSI0j+IUQrDd7+ZtR9WKIGtoYAYAJUKcKhYL
 github.com/quasilyte/go-consistent v0.0.0-20190521200055-c6f3937de18c/go.mod h1:5STLWrekHfjyYwxBRVRXNOSewLJ3PWfDJd1VyTS21fI=
 github.com/quasilyte/go-ruleguard v0.1.2-0.20200318202121-b00d7a75d3d8/go.mod h1:CGFX09Ci3pq9QZdj86B+VGIdNj4VyCo2iPOGS9esB/k=
 github.com/quobyte/api v0.1.2/go.mod h1:jL7lIHrmqQ7yh05OJ+eEEdHr0u/kmT1Ff9iHd+4H6VI=
+github.com/radtriste/kogito-operator v0.0.0-20210406191637-0c6a48be5404 h1:d3xCC5tXl36eqlAeXhSYcn0Y7z1QogPdpj+bO8OummI=
+github.com/radtriste/kogito-operator v0.0.0-20210406191637-0c6a48be5404/go.mod h1:s7IgEpGfN6Blwbsm1nEkGcnQxmSeMWJJeeilfU8cDW8=
```

Comment:
is it a leftover?

Reply:
seems so ... nice catch, gonna remove that